CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+### ⚡ Performance
+- Optimise channel list view updates [#561](https://github.com/GetStream/stream-chat-swiftui/pull/561)
+
```

Comment:
can we also add 2 entries for changes:
- we don't rely on the channel id anymore (some customers might have some logic around this)
- the deprecation of the channel avatar init